### ISSUE TITLE
Error message when encounter REPL comment

### ISSUE THREAD

**Issue by Yujie-W** [timholy/Revise.jl#573]
There is not any problem for comments within functions. But error messages appear for REPL comments. It is probably a bug to report... (maybe duplicated)
```
~ >>> julia                                                                                                                                                                                                 
               _
   _       _ _(_)_     |  Documentation: https://docs.julialang.org
  (_)     | (_) (_)    |
   _ _   _| |_  __ _   |  Type "?" for help, "]?" for Pkg help.
  | | | | | | |/ _` |  |
  | | |_| | | | (_| |  |  Version 1.5.3 (2020-11-09)
 _/ |\__'_|_|_|\__'_|  |  Official https://julialang.org/ release
|__/                   |

julia> # this is a comment

julia> using Revise

julia> # this is a comment
ERROR: MethodError: no method matching unwrap(::Nothing)
Closest candidates are:
  unwrap(::Revise.RelocatableExpr) at /home/username/.julia/packages/Revise/fwStr/src/utils.jl:87
  unwrap(::Expr) at /home/username/.julia/packages/Revise/fwStr/src/utils.jl:71
Stacktrace:
 [1] revise_first(::Nothing) at /home/username/.julia/packages/Revise/fwStr/src/packagedef.jl:1185
 [2] #invokelatest#1 at ./essentials.jl:710 [inlined]
 [3] invokelatest at ./essentials.jl:709 [inlined]
 [4] eval_user_input(::Any, ::REPL.REPLBackend) at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/REPL/src/REPL.jl:132
 [5] repl_backend_loop(::REPL.REPLBackend) at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/REPL/src/REPL.jl:195
 [6] start_repl_backend(::REPL.REPLBackend, ::Any) at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/REPL/src/REPL.jl:180
 [7] run_repl(::REPL.AbstractREPL, ::Any; backend_on_current_task::Bool) at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/REPL/src/REPL.jl:292
 [8] run_repl(::REPL.AbstractREPL, ::Any) at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/REPL/src/REPL.jl:288
 [9] (::Base.var"#807#809"{Bool,Bool,Bool,Bool})(::Module) at ./client.jl:399
 [10] #invokelatest#1 at ./essentials.jl:710 [inlined]
 [11] invokelatest at ./essentials.jl:709 [inlined]
 [12] run_main_repl(::Bool, ::Bool, ::Bool, ::Bool, ::Bool) at ./client.jl:383
 [13] exec_options(::Base.JLOptions) at ./client.jl:313
 [14] _start() at ./client.jl:506
```

**Comment by jaakkor2** [comment]
Revise.jl v3.1.6 seems to work, v3.1.7 and v3.1.8 not.

**Comment by timholy** [comment]
Wow, thanks for reporting this. Looks like I broke this in #563. Fix in #577.
